lib/miquido_lints.yaml

```diff
@@ -0,0 +1,336 @@
+analyzer:
+  plugins:
+    - dart_code_metrics
+
+  # Explanation for this settings are described in Step 3 of this https://dash-overflow.net/articles/getting_started/
+  strong-mode:
+    implicit-casts: false
+    implicit-dynamic: false
+  
+  exclude:
+    - "**/*.g.dart"
+    - "**/*.freezed.dart"
+    - lib/generated/**
+    - generated_plugin_registrant.dart
+
+  language:
+    strict-raw-types: true
+    strict-casts: true
+    strict-inference: true
+
+  errors:
+    missing_required_param: error
+    missing_return: error
+    unawaited_futures: warning
```

Comment:
I think I would leave lint `unawaited_futures` as it is, because in this way we would kinda force people to consciously add `unawaited()` or `ignore comment`, which would explain why they are doing this

Reply:
Or always follow the correct solution which is 
```dart
Future<void> myAsycMethod() async { 
await stuff();
}
```

It is reviewer responsibility to not allow merging suppressed by `ignore` code. That is the reason why lints exist in a first place and we add `flutter analyze` to CI/CD pipelines. This is important rule  and I endorse it to the level that I consider it at warning level.